mizar/daemon/interface_service.py

```diff
@@ -135,14 +133,11 @@ def _ConsumeInterfaces(self, pod_name, cni_params):
         Remove the pod from the interfaces_q and provision the interface
         (program the transit agent)
         """
-        with self.interfaces_lock:
-            if pod_name in self.queued_pods:
-                self.queued_pods.remove(pod_name)
-            interfaces = self.interfaces.get(pod_name, [])
-            for interface in interfaces:
-                if interface.status == InterfaceStatus.queued:
-                    self._ProvisionInterface(interface, cni_params)
-                    interface.status = InterfaceStatus.consumed
+        interfaces = self.interfaces.get(pod_name, [])
+        for interface in interfaces:
+            if interface.status == InterfaceStatus.queued:
+                self._ProvisionInterface(interface, cni_params)
```

Comment:
Does this need to be synchronized? I'm just wondering what the intent of the original mutex was.